weave-js/src/components/PagePanelComponents/Home/Browse3/pages/ChatView/ShowMoreButton.tsx

```diff
@@ -0,0 +1,34 @@
+import {Button} from '@wandb/weave/components/Button';
+import classNames from 'classnames';
+import React, {Dispatch, SetStateAction} from 'react';
+
+type ShowMoreButtonProps = {
+  isShowingMore: boolean;
+  setIsShowingMore: Dispatch<SetStateAction<boolean>>;
+  isUser?: boolean;
+};
+export const ShowMoreButton = ({
+  isShowingMore,
+  setIsShowingMore,
+  isUser,
+}: ShowMoreButtonProps) => {
+  return (
+    <div
+      className={classNames(
+        'absolute bottom-0 left-0 flex w-full items-center justify-center pb-4 pt-20',
+        !isShowingMore
+          ? `from-34% rounded-b-xl bg-gradient-to-t ${
```

Comment:
Is this gradient working?

Reply:
i updated the gradient color to the moon_100 for the new system prompt color and double checked that I could see it for user and assistant